crates/re_viewer_context/src/component_ui_registry.rs

```diff
@@ -7,36 +7,40 @@ use re_types::ComponentName;
 
 use crate::ViewerContext;
 
-/// Controls how mich space we use to show the data in a component ui.
+/// Specifies the context in which the UI is used and the constraints it should follow.
 #[derive(Clone, Copy, Debug, PartialEq, Eq)]
-pub enum UiVerbosity {
-    /// Keep it small enough to fit on one row.
-    Small,
+pub enum UiLayout {
+    /// Display a short summary. Used in lists.
+    ///
+    /// Keep it small enough to fit on half a row. Text should truncate.
```

Comment:
"half a row" ? O.o

Reply:
I refer to the list item's second column here. I'll clarify the comment.